core/src/main/resources/jenkins/CLI/WarnWhenEnabled/message_zh_CN.properties

```diff
@@ -0,0 +1,28 @@
+# The MIT License
+#
+# Copyright 2017 CloudBees, Inc.
```

Comment:
AFAIK you do not work for CloudBees (please correct me if I am wrong) :)
Please use your personal/company copyright in such case

Reply:
Got it.